exonum/src/node/basic.rs

```diff
@@ -21,15 +21,11 @@ use std::error::Error;
 
 use messages::{Any, RawMessage, Connect, Status, Message, PeersRequest};
 use helpers::Height;
-use events::Channel;
-use super::{NodeHandler, RequestData, ExternalMessage, NodeTimeout};
+use super::{NodeHandler, RequestData};
 
-impl<S> NodeHandler<S>
-where
-    S: Channel<ApplicationEvent = ExternalMessage, Timeout = NodeTimeout>,
-{
+impl NodeHandler {
     /// Redirects message to the corresponding `handle_...` function.
-    pub fn handle_message(&mut self, raw: RawMessage) {
+    pub fn handle_message(&mut self, _peer: SocketAddr, raw: RawMessage) {
```

Comment:
Why `SocketAddr` is needed here? Will it be used in the future? Please remove it or add a comment about its future usage.

Reply:
I am pretty sure we should carry some `connection context` but as for me `SocketAddr` is a bad `handle`

Reply:
Any update on this?